recipes-azure-iot/azure-identityd/aziot-identityd_1.4.7.bb

```diff
@@ -271,7 +271,7 @@ LIC_FILES_CHKSUM = " \
     "
 
 SUMMARY = "The code used for Identity Service."
-HOMEPAGE = "git@github.com:Azure/iot-identity-service.git"
+HOMEPAGE = ""
```

Comment:
https://github.com/Azure/iot-identity-service ?

Reply:
or remove the line completely?

Reply:
this is generated code. it depends whatever you have set as homepage in `identity/aziot-identityd/Cargo.toml` in the iot-identity-service repo, when generating it. unfortunately upstream doesn't set it. without setting it `cargo bitbake` fails.